pkg/asset/targets/targets.go

```diff
@@ -30,7 +29,6 @@ var (
 		&machines.ClusterAPI{},
 		&manifests.Manifests{},
 		&manifests.Openshift{},
-		&clusterapi.Cluster{},
```

Comment:
We don't want to remove this

Reply:
You said to remove it here: https://github.com/openshift/installer/pull/8018#discussion_r1526873587

Does it stay, or does it go? ;-)

Reply:
Added it back.